backend/app/common/protocol_conversion.py

```
@@ -0,0 +1,582 @@
+
+    if request_protocol == ANTHROPIC_PROTOCOL and supplier_protocol == OPENAI_PROTOCOL:
+        if _HAS_EXPERIMENTAL_PASSTHROUGH:
+            model_response = ModelResponse(**body)
+            translated = AnthropicExperimentalPassThroughConfig().translate_openai_response_to_anthropic(  # type: ignore[union-attr]
+                response=model_response
+            )
+            return translated.model_dump(exclude_none=True)
+        return _translate_openai_response_to_anthropic(body, target_model)
+
+    if request_protocol == OPENAI_PROTOCOL and supplier_protocol == ANTHROPIC_PROTOCOL:
+        dummy_logger = type("DummyLogger", (), {"post_call": lambda *args, **kwargs: None})()
```

Ruff (0.14.10)

Unused lambda argument: `args`

(ARG005)